src/Bookshelf.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Book from "./Book";
+
+class Bookshelf extends React.Component {
+  constructor(props) {
+    super();
```

Comment:
and if used, super should be called with props, I use eslint rules for this also.